interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX lets us code HTML in React. It is an extension of JavaScript. One huge difference between JSX and HTML is that JSX uses "className" instead of "class" when it defines class attributes. Also, JSX has props and HTML has attributes.
 
-Researched answer:
+Researched answer: JSX is an extension of JavaScript that lets us write HTML directly within JavaScript. One key difference between JSX and HTML is JSX uses camelCase for attributes and event references.
 
 1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yarn is a tool and package manager that changes yarn.lock and node_modules files within the entire project.  
 
-Researched answer:
+Researched answer: Yarn is a package manager that used to install pieces of code that serve a particular purpose. It downloads multiple packages at the same time (rather than individually), which speeds up the installation process. It modifies the yarn.lock and node_modules. 
 
 3. What is a React component?
 
-Your answer:
+Your answer: A React component is similar to a puzzle piece. It helps the website operate the way that the developer intends for it to operate. There are function and class components. Functional components are simple and hold a small amount of data; however, class components are more complex and change over time. 
 
-Researched answer:
+Researched answer: A React component is a building block that can be recognized as a JavaScript function that returns a React element. There are function components that are simple and concise, and there are class components that can hold state.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: The state values represents the internal data of a component, like the user input. It usually contains the data that will continue to change over time. It is like a variable. Props are similar to arguments that are passed into a component. They cannot be changed within the component.  
 
-Researched answer:
+Researched answer: Props are can be used to pass data from one component to another; however, they cannot be modified. State values can change; each component has a state.
 
 5. What is the DOM?
 
-Your answer:
+Your answer: The DOM (Document Object Model) is an interface that allows us to manipulate the HTML properties of a web page. 
 
-Researched answer:
+Researched answer: The DOM is a tree-like structured representation of all of the HTML elements on a web-page or application.
```

Comment:
This would be a great place to add more content! Like do you know of any libraries or frameworks that use the DOM?

Reply:
I definitely should have gone more in detail